components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -67,10 +69,33 @@ function CustomLeague:customParseArguments(args)
 	self.data.publishertier = tostring(Logic.readBool(args.featured))
 	self.data.status = self:_getStatus(args)
 
+	self.data.startTime = self:_readStartTime()
```

Comment:
```suggestion
	self.data.startTime = Logic.wrapTryOrLog(self:_readStartTime)()
```
Don't want to ruin the entire infobox just because the time was invalid